cns/service.go

```diff
@@ -316,3 +327,20 @@ func (service *Service) SendErrorResponse(w http.ResponseWriter, errMsg error) {
 	err := acn.Encode(w, &resp)
 	logger.Errorf("[%s] %+v %s.", service.Name, &resp, err.Error())
 }
+
+// TLSVersionNumber returns the version number for the provided TLS version name
+// (e.g. 0x0301)
+func TLSVersionNumber(versionName string) (uint16, error) {
+	switch versionName {
+	case "TLS 1.0":
+		return tls.VersionTLS10, nil
+	case "TLS 1.1":
+		return tls.VersionTLS11, nil
```

Comment:
since we already support a min TLS version of 1.2 as the hardcoded minimum, let's not support any TLS versions lesser than this. this will align with Azure's TLS version guidance as well for their services.

Reply:
Yeah +1 to that. Easier to prevent a downgrade attack by just flat-out saying "no" to older TLS versions.